src/controllers/donation.controller.ts

```
@@ -0,0 +1,65 @@
+export const fileMatchingRequest = async (req: Request, res: Response) => {
+  try {
+    const { donationId } = req.params;
+    const donation = await Donation.findById(donationId);
+    if (!donation || !donation.employerName) {
+      return res.status(404).json({ success: false, error: 'Donation or employer info not found' });
+    }
+
+    const result = await CorporateMatchingService.autoFileMatchingRequest(
+      donation._id.toString(),
+      donation.employerName,
+      '12-3456789', // Club Tax EIN reference from #4667
```

ESLint

[error] Replace `'12-3456789'` with `"12-3456789"`

(prettier/prettier)